tensorflow/python/ops/nn_impl.py

```diff
@@ -1396,7 +1396,7 @@ def fused_batch_norm(
   min_epsilon = 1.001e-5
   epsilon = epsilon if epsilon > min_epsilon else min_epsilon
 
-  if compat.forward_compatible(2019, 6, 6):
+  if compat.forward_compatible(2019, 11, 10):
```

Comment:
Let's make this 11/11 like other places.

Reply:
@penpornk  Thanks for the review.   Since the test moves the compatibility horizon to a later date for testing, we set at 1 day past the date the compatibility at the test, This similar to the other tests that were it is set to 6/7 while the feature becomes active on 6/6  Let me know if I need to change, I will also try 11/11 here and see if it works.